benchmarks/throw.rb

```diff
@@ -0,0 +1,39 @@
+require 'harness'
+
+def foo
+  throw 'error'
```

Comment:
This is a different `throw` from what we're talking about. This is an `opt_send_without_block` instruction, not `throw` insn.

If you look at liquid's example code that I shared in https://github.com/Shopify/ruby/issues/504#issuecomment-1462688863 ([example](https://github.com/Shopify/liquid/blob/48cb643c026557f48e524dfd39cc9ff90aa3db95/lib/liquid/context.rb#L247)), it's a `return` inside a block, which cannot be `leave` instruction because it does make a cross-instruction jump from the block to the method.